README.txt

```diff
@@ -23,8 +23,9 @@ You will need to install the following software to play Endgame: Singularity:
  * Python 3 (https://python.org/download/)
  * pygame (https://www.pygame.org/download.shtml)
  * NumPy (https://www.scipy.org/install.html)
+ * unidecode (https://pypi.org/project/Unidecode/)
```

Comment:
I would prefer that we use the built-in `locale.strxfrm` (or its counterpart `locale.strcoll`) by default - possibly ICU as an _optional_ alternative, which is allegedly the only truly reliable way to sort strings locale-aware (notably, there is trouble with `locale.strxfrm` on Mac OS).

Example of how to use ICU is at: https://stackoverflow.com/questions/11121636/sorting-list-of-string-with-specific-locale-in-python

Reply:
Thanks for those libraries - I found this difficult to search for, since must search results for "lexicographic sorting" will just give you sorting by unicode code point, which is fine for English.

Reply:
I have switched to ICU. Note that none of the libraries (I tested all of them) can handle correct sorting order for German umlauts, so I have added some special code for that.

Reply:
Thanks for looking at it.  It looks like ICU is now mandatory rather than optional as I had hoped for.  That is the only bit that is missing for me. :)

Reply:
I have added the packaging info now - I already had ICU installed, so I didn't notice.

This branch can't be tested right now because I merged master, which is broken.